lerobot/scripts/train.py

```diff
@@ -106,40 +104,38 @@ def update_policy(
     grad_clip_norm,
     grad_scaler: GradScaler,
     lr_scheduler=None,
-    use_amp: bool = False,
+    accelerator=None,
 ):
     """Returns a dictionary of items for logging."""
     start_time = time.perf_counter()
-    device = get_device_from_parameters(policy)
+
     policy.train()
-    with torch.autocast(device_type=device.type) if use_amp else nullcontext():
-        output_dict = policy.forward(batch)
-        # TODO(rcadene): policy.unnormalize_outputs(out_dict)
-        loss = output_dict["loss"]
-    grad_scaler.scale(loss).backward()
```

Comment:
the unscaling is handled in the `accelerator.clip_grad_norm_`, the scaling in the `backward`, the step and update in `optimizer.step`: 
https://github.com/huggingface/accelerate/blob/e1247de01e0733c5d21075cb6f39b2605f4be123/src/accelerate/accelerator.py#L2255

Reply:
If `backward` handles the scaling, and gradient clipping handles the unscaling, what if we don't use gradient clipping? Where is the unscaling handled then?

Reply:
in our previous training script we also always called clip_grad_norm, so I'm not sure about how the usecase in which it could happen 🤔 do we want to make gradient clipping optional?

Reply:
Hm well I'd say 2 separate things here:

1. I think gradient clipping should be optional in general, but I wouldn't add that idea to this PR. In saying so, maybe we should account for the possibility that we want to make it optional in a later PR.
2. I'm not comfortable with not understanding the answer to this question. ie, how would unscaling be handled if we were not to do gradient clipping. It seems strange to me that unscaling happens implicitly in an optional step. I'd be happier approving the PR if we understand.

Reply:
You're right that we should distinguish these two processes. I will update this.

Reply:
done. 
fyi: `accelerator.unscale() `is conditional on whether mixed precision is used : 
https://github.com/huggingface/accelerate/blob/e1247de01e0733c5d21075cb6f39b2605f4be123/src/accelerate/accelerator.py#L2220

Reply:
Nice!